tests/helpers.py

```diff
@@ -35,6 +35,11 @@ def get_reference_font(fname, fsize=12):
     return ImageFont.truetype(path, fsize)
 
 
+def get_reference_pil_font(fname):
+    path = get_reference_file(os.path.join('font', fname))
```

Comment:
missing docstring.

Reply:
and should use 'get_reference_pillow_font' instead imo.

Reply:
Added

Reply:
I don't see the change though, did you push the changes to the branch?

Reply:
Will push all changes shortly.